vault/resource_ldap_secret_backend.go

```diff
@@ -0,0 +1,684 @@
+		}
+	}
+	if val, ok := resp.Data["last_rotation_tolerance"]; ok {
+		if err := d.Set("last_rotation_tolerance", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'last_rotation_tolerance': %s", err))
+		}
+	}
+	if val, ok := resp.Data["length"]; ok {
+		if err := d.Set("length", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'length': %s", err))
+		}
+	}
+	if val, ok := resp.Data["max_ttl"]; ok {
+		if err := d.Set("max_ttl", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'max_ttl': %s", err))
+		}
+	}
+	if val, ok := resp.Data["password_policy"]; ok {
+		if err := d.Set("password_policy", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'password_policy': %s", err))
+		}
+	}
+	if val, ok := resp.Data["request_timeout"]; ok {
+		if err := d.Set("request_timeout", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'request_timeout': %s", err))
+		}
+	}
+	if val, ok := resp.Data["starttls"]; ok {
+		if err := d.Set("starttls", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'starttls': %s", err))
+		}
+	}
+	if val, ok := resp.Data["tls_max_version"]; ok {
+		if err := d.Set("tls_max_version", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'tls_max_version': %s", err))
+		}
+	}
+	if val, ok := resp.Data["tls_min_version"]; ok {
+		if err := d.Set("tls_min_version", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'tls_min_version': %s", err))
+		}
+	}
+	if val, ok := resp.Data["ttl"]; ok {
+		if err := d.Set("ttl", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'ttl': %s", err))
+		}
+	}
+	if val, ok := resp.Data["upndomain"]; ok {
+		if err := d.Set("upndomain", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'upndomain': %s", err))
+		}
+	}
+	if val, ok := resp.Data["url"]; ok {
+		if err := d.Set("url", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'url': %s", err))
+		}
+	}
+	if val, ok := resp.Data["use_pre111_group_cn_behavior"]; ok {
+		if err := d.Set("use_pre111_group_cn_behavior", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'use_pre111_group_cn_behavior': %s", err))
+		}
+	}
+	if val, ok := resp.Data["use_token_groups"]; ok {
+		if err := d.Set("use_token_groups", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'use_token_groups': %s", err))
+		}
+	}
+	if val, ok := resp.Data["userattr"]; ok {
+		if err := d.Set("userattr", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'userattr': %s", err))
+		}
+	}
+	if val, ok := resp.Data["userdn"]; ok {
+		if err := d.Set("userdn", val); err != nil {
+			return diag.FromErr(fmt.Errorf("error setting state key 'userdn': %s", err))
+		}
+	}
```

Comment:
Similar comment as above. We can make a list of API fields and loop over them as done [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_kubernetes_secret_backend_role.go#L200-L218)